src/app/core/services/filters.service.ts

```diff
@@ -4,17 +4,30 @@ import { BehaviorSubject, pipe } from 'rxjs';
 
 export type Filter = {
   title: string;
-  status: string;
+  status: string[];
   type: string;
   sort: Sort;
   labels: string[];
   milestones: string[];
   hiddenLabels?: Set<string>;
 };
 
+type StatusInfo = {
+  type: string;
+  status: string;
+};
+
+/**
+ * Converts a status string into an info object
+ */
+export const statusToType = (statusString: string): StatusInfo => {
+  const [status, type] = statusString.split(' ');
+  return { status, type };
+};
+
 export const DEFAULT_FILTER: Filter = {
   title: '',
-  status: 'all',
+  status: ['open pullrequest', 'merged pullrequest', 'closed pullrequest', 'open issue', 'closed issue'],
```

Comment:
As per the description of #296, ``closed pullrequest`` should not be selected by default

Reply:
Fixed!